peer_mentoring/tests/fixtures.py

```diff
@@ -0,0 +1,99 @@
+import io
```

Comment:
you can also put these (great work btw on this setup code!) in conftest.py and they become "discoverable" automatically by pytest

Reply:
Just fixed with the latest push, but forgot to delete this file. Thanks for the feedback.